css/css-values/calc-mixed-units.html

```diff
@@ -0,0 +1,31 @@
+<html lang="en">
+  <head>
+    <style>
+      .box {
+        background: rgb(0, 0, 255);
+        width: 100vw;
+        height: 10px;
+      }
+
+      @media (min-width: calc(100vw - 1em)) {
```

Comment:
I think you should be able to come up with a much stricter test here.  If you put a test inside an iframe with a known size, and assume the default `16px` font size, you should be able to test queries for `width` and `height` exactly without using either `min-` or `max-`.  A few of those (different combinations of `vw`, `em`, and `px`, at least) would seem to be a somewhat comprehensive test for "calc-mixed-units", whereas passing this test alone doesn't ensure very much.